pkg/apis/internal.admin.acorn.io/v1/quotarequests.go

```diff
@@ -0,0 +1,70 @@
+package v1
+
+import (
+	v1 "github.com/acorn-io/acorn/pkg/apis/internal.acorn.io/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/strings/slices"
+)
+
+const (
+	QuotaRequestCondition          = "quota-request"
+	ProjectEnforcedQuotaAnnotation = "acorn.io/enforced-quota"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type QuotaRequestInstance struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	SupportedRegions []string                   `json:"supportedRegions,omitempty"`
```

Comment:
Apps only support one region, so I think these should, too.
```suggestion
	Region string                   `json:"supportedRegions,omitempty"`
```

Reply:
Makes sense. Updated the region functions to reflect this change as well.